### PR TITLE
honkerative MODsuit theme, and paint-kit!

### PR DESCRIPTION
**Pull request by capsaicinz**
## About The Pull Request

you've all been waiting for it- clown operative MODsuits! clown operatives gain access to a MODsuit paint kit free of charge, and can use this to turn their boring old syndicate MODsuits into funny, clown-styled ones! this has no mechanical change whatsoever, aesthetics only.

unknown

## Why It's Good For The Game

makes clown operatives slightly more distinguishable (and funny) from nuclear operatives.

## Changelog

:cl:
add: Honk Co. is proud to announce their new project: clown operative MODsuits! Adds the "Honkerative" theme to Syndicate MODsuits, and a paint kit module to clown operative uplinks. Honka honka.
/:cl:

### CODE REVIEW

**Comment by capsaicinz** [comment]
aside from that mess there, @LemonInTheDark please label this with DO NOT MERGE until the modsuit files are reorganized, which should be happening in the very very near future..

**Comment by LemonInTheDark** [comment]
I believe those are in a merge freeze regardless, so unless a rouge maintainer fucks you over you should be fine
if they do, s their problem not yours

**Comment by capsaicinz** [comment]
COUGH COUGH actually, it's way sooner than i intended, so _no need for that_. just need to wait until it's merged hopefully by tonight.

**Comment by capsaicinz** [comment]
> I believe those are in a merge freeze regardless, so unless a rouge maintainer fucks you over you should be fine if they do, s their problem not yours

thumbsu uppu

File: code/modules/mod/mod_types.dm [hunk 1]
```diff
@@ -158,6 +158,18 @@
...
+	applied_skin = "honkerative"
+	theme = /datum/mod_theme/syndicate
+	cell = /obj/item/stock_parts/cell/hyper
+	initial_modules = list(
+		/obj/item/mod/module/storage/syndicate,
+		/obj/item/mod/module/welding,
+		/obj/item/mod/module/jetpack,
+		/obj/item/mod/module/flashlight,
+		/obj/item/mod/module/holster,
+		)
```
Comment (Fikou) [inline code/modules/mod/mod_types.dm:171]: put this at the bottom and make it an initialize_immediate skin instead, and give it to prefs menu clown outfits

**Review by Ghommie — CHANGES_REQUESTED** [review]
It seems you are accidentally deleting the paper_maker icon state in the objs/mod.dmi file with these changes. Please fix that, thank you.

File: code/modules/mod/mod_types.dm [hunk 2]
```diff
@@ -303,3 +303,9 @@
 	theme = /datum/mod_theme/elite
 
 INITIALIZE_IMMEDIATE(/obj/item/mod/control/pre_equipped/syndicate_empty)
+
+/obj/item/mod/control/pre_equipped/honkerative
+	applied_skin = "honkerative"
+	theme = /datum/mod_theme/syndicate
+
+INITIALIZE_IMMEDIATE(/obj/item/mod/control/pre_equipped/honkerative)
```
Comment (Fikou) [inline code/modules/mod/mod_types.dm:311]: actually id just make this a subtype of syndicate empty since its a skin and itd inherit initialize immediate

**Comment by capsaicinz** [comment]
unknown

**Pull request merged**